ckan/common.py

```diff
@@ -10,7 +10,7 @@
 from __future__ import annotations
 
 import logging
-from collections import MutableMapping
+from collections.abc import MutableMapping
```

Comment:
If this was broken is it unused? can it be removed?

Reply:
It works, but an import from `collections` is deprecated since python 3.7 and it's recommended to import from `collections.abc` instead. This class is used as a superclass for the config object.